test/integration/tests/pcv.ts

```diff
@@ -232,9 +232,9 @@ describe('e2e-pcv', function () {
         params: [aaveEthPCVDeposit.address]
       });
 
-      await aaveEthPCVDeposit.withdrawERC20(await aaveEthPCVDeposit.aToken(), deployAddress, tenPow18.mul(toBN(10000)));
```

Comment:
Reduced all figures in this test by 10x, to account for the balance of the on-chain Aave PCV deposit having fewer funds